### PR TITLE
23qQ4-open-source-infrastructure-engineer post

### PR DESCRIPTION
**Pull request by damianavila**
This is the job posting based on https://docs.google.com/document/d/1Jze2Pgt_pVxgFVgUVEtjS5dwL2-k9FhSCGng7Y7Fqx0/edit. I intend to publish it by Tuesday (tomorrow) EOD unless there is a blocker from someone.

### CODE REVIEW

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
++++
+authors =  ["Damián Avila"]
+title = "Open Source Infrastructure Engineer"
+work_description = [
```
Comment (damianavila) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md:4]:
```suggestion
work_description = [
  "Cloud infrastructure management and operations",
  "Site Reliability Engineering",
  "Development of open source infrastructure for hosted JupyterHub service"
]
```

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+to cloud-based resources for research and education purposes. This engineer will be part of an awesome
+engineering team pushing forward the development and reliable operations of our cloud-based infrastructure.
+
+<!-- Defined in layouts/shortcodes/job_details.html -->
+{{% job_details %}}
+
+## Who we are
+
+2i2c is a non-profit organization with a mission to make interactive computing more accessible,
+scalable, and powerful for research and education: [https://2i2c.org/about/](https://2i2c.org/about/).
```
Comment (damianavila) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
scalable, and powerful for research and education: [https://2i2c.org/mission/](https://2i2c.org/mission/).
```

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+authors =  ["Damián Avila"]
+title = "Open Source Infrastructure Engineer"
+work_description = [
+  "Cloud infrastructure management and operations",
+  "Site Reliability Engineering",
+  "Development of open source infrastructure for hosted JupyterHub service"
+]
+salary_range = "$121,600 - $130,500"
+location = "Remote, required overlapping for US Central and US Pacific timezones"
+date = "2022-10-09"
```
Comment (damianavila) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
date = "2022-10-10"
```

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+Please fill out the [following form](https://forms.gle/k75VmYUosxuyPWfi7).
+
+## Final note
+
+We are looking for engineers with working hours overlapping US Central and US Pacific timezones.
+This is an essential requirement to resolve an existing imbalance in the team (most of our engineers
+are located in European timezones). At this time, we will only review applications compatible with
+that essential requirement.
+
+Keep in mind we will begin reviewing applications around Oct 17th, and will accept them on
```
Comment (damianavila) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
Keep in mind we will begin reviewing applications around Oct 16th, and will accept them on
```

File: content/jobs/2023/technical-content-specialist.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ location = "Fully remote"
 url_apply = "https://forms.gle/H43CsVUoF7e2C2Nm8"
 deadline = "We will begin reviewing applications around **August 2, 2023**, and will accept them on a rolling basis until the position is filled."
 date = "2023-07-12"  # This is the date we _post_ this job.
-open = true  # SET TO FALSE IF THE JOB IS NO LONGER OPEN
+open = false  # SET TO FALSE IF THE JOB IS NO LONGER OPEN
```
Comment (damianavila) [inline content/jobs/2023/technical-content-specialist.md:13]: Unrelated to the eng job post but confirmed by @jmunroe via Slack.

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+## Essential requirements
+
+- Experience with deploying applications on cloud infrastructure.
+- Experience deploying and developing with Linux container-based technologies, such as Docker and Kubernetes.
+- Experience with continuous integration services (e.g. Circle CI, GitHub workflows).
+- Experience developing tools in a general purpose programming language (eg. Python).
+- Experience collaborating and coordinating work via online platforms, such as GitHub, GitLab, or BitBucket,
+  and distributed revision control.
+- Experience working with distributed service teams that use asynchronous methods of communication
+- Working hours that overlap with US Central and US Pacific time zones
```
Comment (yuvipanda) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
- Working hours that overlap with US Pacific time zones
```

I'd prefer we specifically say US Pacific

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+You will also find relevant information about our compensation philosophy and our commitment to equal
+opportunity. For an even deeper look into how 2i2c works, check out our [Team Compass](https://compass.2i2c.org/).
+
+## How to apply
+
+Please fill out the [following form](https://forms.gle/k75VmYUosxuyPWfi7).
+
+## Final note
+
+We are looking for engineers with working hours overlapping US Central and US Pacific timezones.
```
Comment (yuvipanda) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
We are looking for engineers with working hours overlapping US Pacific timezones.
```

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+work we do, the communities we serve, and our own organizational culture.
+
+2i2c is a 501(c)(3) not-for-profit project fiscally sponsored by Code for Science and Society.
+
+
+## What you’ll do
+
+- Ensures the reliable operation of the 2i2c infrastructure (leveraging production-ready cloud-based
+  tools such as JupyterHub, BinderHub, and Dask).
+- Attends support-facing tickets
```
Comment (yuvipanda) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]: I feel we should expand this a little bit more and clarify that this isn't their *primary* role, but they will be part of a user facing support rotation.

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+- Explores emerging technologies in the Cloud / DevOps space, design and implement cloud computing
+  architecture in partnership with our team.
+- Participates in upstream open source communities we rely on (such as JupyterHub, BinderHub, Dask, etc)
+  in partnership with the established leaders of those communities and collaborate with the Community Lead
+  in the education and outreach around cloud computing.
+- Work with a distributed and global team - team members are given a lot of autonomy, and expected to be
```
Comment (yuvipanda) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md:57]: We should clarify that we do *not* expect people to carry a pager.

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md (hunk 8 above)
Comment (damianavila) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
- Attends support-facing tickets as part of a support rotation process/group (this is not your _primary_ task, although it is currently a shared load among engineers).
```

Comment (damianavila) [reply content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]: I tried to clarify it, we can change it after it is published if it is not enough (or if there is a better version).

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+- Ensures the reliable operation of the 2i2c infrastructure (leveraging production-ready cloud-based
+  tools such as JupyterHub, BinderHub, and Dask).
+- Attends support-facing tickets as part of a support rotation process/group (this is not your _primary_ task, although it is currently a shared load among engineers).
+- Explores emerging technologies in the Cloud / DevOps space, design and implement cloud computing
+  architecture in partnership with our team.
+- Participates in upstream open source communities we rely on (such as JupyterHub, BinderHub, Dask, etc)
+  in partnership with the established leaders of those communities and collaborate with the Community Lead
+  in the education and outreach around cloud computing.
+- Work with a distributed and global team - team members are given a lot of autonomy, and expected to be
+  proactive at communicating with one another and working with others to allocate effort that will maximize
```
Comment (damianavila) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
  proactive (we do not expect people to carry a pager) at communicating with one another and working with others to allocate effort that will maximize
```

Comment (damianavila) [reply content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]: Added the clarification.

**Comment by damianavila** [comment]
Addressed the comments and I going to merge it after checking the final version.
Also updated the other documents in the dedicated folder to ask for US Pacific time zone overlapping.

File: content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
++++
+authors =  ["Damián Avila"]
+title = "Open Source Infrastructure Engineer"
+work_description = [
+  "Cloud infrastructure management and operations",
+  "Site Reliability Engineering",
+  "Development of open source infrastructure for hosted JupyterHub service"
+]
+salary_range = "$121,600 - $130,500"
+location = "Remote, required overlapping for US Central and US Pacific timezones"
```
Comment (damianavila) [inline content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md]:
```suggestion
location = "Remote, required overlapping for US Pacific timezones"
```

**Pull request merged**